less/index.less

```diff
@@ -52,8 +52,345 @@ table {
+                        padding-bottom: 7px;
+                    }
+                    table {
+                        width: 100%;
+                    }
+                    tr {
+                        border-top: 1px solid @clr-light-grey;
+                        font-size: 1.1rem;
+                        &:first-child {
+                            font-weight: bold;
+                            font-size: 1.4rem;
+                            border-top: none;
+                            border-bottom: 2px solid @clr-light-grey;
+                        }
+                        &:nth-child(even) {
+                            background-color: rgba(22, 22, 22, .1);
+                        }
+                        td {
+                            padding: 25px 5px;
+                            &:first-child {
+                                font-weight: bold;
+                            }
+                        }
+                    }
+                }
+
+                /*bottom-content */
+                #contact {
+                    display: flex;
+                    flex-wrap: wrap;
+                    margin: 20px 0px 15px;
+                    h3 {
+                        margin-bottom: 20px;
+                    }
+                    form {
+                        .column-left;
+                        width: 300px;
+                        input, textarea {
+                            padding: 10px;
+                            border: 1.5px solid CornflowerBlue;
+                        }
+                        input {
+                            margin-bottom: 20px;
+                        }
+                    }
+                    .btn {
+                        padding: 5px 49px;
+                        margin: 16px 0px 0px;
+                        border: 1.5px solid #2e2e2e;
+                        background-color: #2e2e2e;
+                        color: #fff;
+                        font-size: 1.6rem;
+                        font-weight: bold;
+                    }
+                    .social {
+                        .column-left;
+                        // align-items: flex-start;
+                        ul {
+                            width: 100%;
+                            align-items: flex-start;
+                            li {
+                                padding: 10px 0px;
+                                a {
+                                    text-decoration: none;
+                                    font-size: 20px;
+                                    color: @clr-black;
+                                    .fab, .fa {
+                                        font-size: 24px;
+                                        padding-right: 15px;
+                                    }
+                                }
+                            }
+                        }
+                    }
+                }
+            }
```

Comment:
Rule of thumb: Never nest more than 4 levels deep. This is because the deeper you go, the more memory it takes to load. Fewer selectors are more performant. 4 Levels Deep is what I was taught while going through the program and it has served me well. Let me know if you have any questions about organizing your stylings, with those limitations in place.